app/indexers/embargo_metadata_indexer.rb

```diff
@@ -1,27 +1,33 @@
 # frozen_string_literal: true
 
 class EmbargoMetadataIndexer
-  attr_reader :resource
+  attr_reader :cocina
 
-  def initialize(resource:, **)
-    @resource = resource
+  def initialize(cocina:, **)
+    @cocina = cocina
   end
 
   # These fields are used by the EmbargoReleaseService in dor-services-app
   # @return [Hash] the partial solr document for embargoMetadata
   def to_solr
-    {
-      'embargo_status_ssim' => embargo_status
-    }.tap do |solr_doc|
-      solr_doc['embargo_release_dtsim'] = Array(release_date.first.utc.strftime('%FT%TZ')) if release_date.first.present?
+    {}.tap do |solr_doc|
+      embargo_release_date = embargo_release_date(cocina)
+      if future?(embargo_release_date)
+        solr_doc['embargo_status_ssim'] = ['embargoed']
+        solr_doc['embargo_release_dtsim'] = [embargo_release_date.utc.strftime('%FT%TZ')]
+      end
     end
   end
 
-  # rubocop:disable Lint/UselessAccessModifier
   private
 
-  # rubocop:enable Lint/UselessAccessModifier
+  def embargo_release_date(cocina)
+    cocina.access.embargo.releaseDate if cocina.access.embargo&.releaseDate.present?
+  end
+
+  def future?(embargo_release_date)
```

Comment:
I think this check could potentially be harmful, because dor-services-app relies on this index so it can find any items that have a past embargo and then lift them.https://github.com/sul-dlss/dor-services-app/blob/main/app/services/embargo_release_service.rb#L8

Reply:
ok - so your recommendation is to put them in the index whenever they are present?

Reply:
Yes.

Reply:
And there will be a dependency on this change: https://github.com/sul-dlss/dor-services-app/pull/2589

Reply:
ok.  the discussion ticket is probably why I wasn't indexing the ones in the past in any case.

Reply:
the change you asked for here ... means I will index any embargo data found in cocina, for future or past embargos.   So sul-dlss/dor-services-app#2589 is irrelevant to this PR.

Reply:
The change you have here says, "any embargo that is past has been released" and that's not a true statement.  An embargo gets released when it gets released, and then (after sul-dlss/dor-services-app#2589), it will no longer show up in the record.

Reply:
fixed - all embargo dates encountered are now treated equally.  We have a zero tolerance policy against discrimination.